juniper_official/Solutions/EVPN-VXLAN/evpn-vxlan.playbook

```diff
@@ -48,13 +48,26 @@
  *     anomalies when exceeds threshold.
  * 23) Rule vxlan-vport.rule, monitors vport count and notifies anomalies
  *     when exceeds threshold.
+ * 24) Rule evpn-db-state-netconf.rule, Verify evpn database state has duplicate
```

Comment:
Same comments as "juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook"

Reply:
updated in the playbook